packages/textlint-config/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@strv/textlint-rule-preset-strv",
+  "description": "STRV's rule preset for textlint",
+  "version": "1.0.0",
+  "author": "Korneliusz Dubieniecki <lekterable@gmail.com>",
+  "bugs": {
+    "url": "https://github.com/strvcom/code-quality-tools/issues"
+  },
+  "contributors": [],
+  "dependencies": {
+    "textlint": "^11.2.5",
+    "textlint-filter-rule-comments": "^1.2.2",
+    "textlint-rule-apostrophe": "^1.0.0",
+    "textlint-rule-common-misspellings": "^1.0.1",
+    "textlint-rule-stop-words": "^1.0.13",
+    "textlint-rule-terminology": "^1.1.30",
+    "textlint-rule-title-case": "^1.0.0",
+    "textlint-rule-write-good": "^1.6.2"
+  },
+  "keywords": [
+    "config",
+    "textlint",
+    "textlintrule",
+    "strv"
+  ],
+  "license": "BSD-3-Clause",
+  "main": "index.js",
```

Comment:
This is redundant, _index.js_ is automatically a main entrypoint for a package. Please remove the field.